docs/installation/metadata-types.rst

```diff
@@ -5,15 +5,16 @@ A Metadata Type defines which fields should be searchable in your product or dat
 
 Three metadata types are added by default called ``eo``, ``telemetry`` and ``eo3``.
 
+You can see the default metadata types in the repository at `datacube/index/default-metadata-types.yaml <https://github.com/opendatacube/datacube-core/blob/develop/datacube/index/default-metadata-types.yaml>`_.
+
 You would create a new metadata type if you want custom fields to be searchable for your products, or
 if you want to structure your metadata documents differently.
 
-You can see the default metadata type in the repository at `datacube/index/default-metadata-types.yaml <https://github.com/opendatacube/datacube-core/blob/develop/datacube/index/default-metadata-types.yaml>`_.
-
 To add or alter metadata types, you can use commands like: ``datacube metadata add <path-to-file>``
 and to update: ``datacube metadata update <path-to-file>``. Using ``--allow-unsafe`` will allow
 you to update metadata types where the changes may have unexpected consequences.
 
+Note that from version 1.9 onward, only eo3-compatible metadata types will be accepted.
```

Comment:
From version 2.0 onwards.

The postgres driver (currently aka "default", from 1.9 aka "legacy") will continue to support non-eo3-compatible metadata types until it is retired in 2.0.

The postgis driver (currently aka "experimental") already only supports non-eo3-compatible metadata types.

Reply:
Fixed